src/storage/txn/scheduler.rs

```diff
@@ -729,15 +729,8 @@ impl<E: Engine, L: LockManager> Scheduler<E, L> {
             }) => {
                 SCHED_STAGE_COUNTER_VEC.get(tag).write.inc();
                 match ctx.get_disk_full_opt() {
```

Comment:
The match seems not necessary. Why not `to_be_write.disk_full_opt = ctx.get_disk_full_opt()`?

Reply:
No, the to_be_write setting will be overwriten by client setting.